### DISCUSSION TITLE
How do the docs achieve this permalink icon?

### DISCUSSION THREAD

**Discussion by harry643** [facelessuser/pymdown-extensions#1355] [Q&A] [answered]
This is an icon called material-link from the Material icon set. In mkdocs.yml it just says `permalink: ""`. Is this icon being added in custom CSS somewhere?

![Screenshot 2021-06-04 at 22 36 59](https://user-images.githubusercontent.com/8885267/120865274-3f637a80-c57d-11eb-9fe2-a3a10273451e.png)

**Comment by facelessuser** [comment, marked as answer]
1. By leaving the link icon blank via the TOC extension: https://github.com/facelessuser/pymdown-extensions/blob/main/mkdocs.yml#L73
2. By providing CSS that adds the inlined image. https://github.com/facelessuser/pymdown-extensions/blob/main/docs/src/scss/extensions/_toc.scss

    We use SCSS to inline the image into the final CSS, but you don't have to. You could just as easily use some icon font or reference the image via a CDN, etc.

    Icon comes from: https://materialdesignicons.com/

**Comment by johnmerlie** [comment]
Found this while trying to answer the same question.

I was able to produce similar output by adding `&#xFE0E;` in my toc.permalink setting based on the answer [here](https://en.wikipedia.org/wiki/Variant_form_(Unicode)#Variation_Selectors_block).

mkdocs.yml
```yaml
markdown_extensions:
  - toc:
      permalink: "🔗&#xFE0E;"
```

Rendered:

![image](https://user-images.githubusercontent.com/40776927/202829077-28dcc30a-534f-4e1e-b835-31ebc21d6c64.png)
